src/Monolog/Handler/SlackHandler.php

```diff
@@ -169,6 +169,15 @@ private function buildContent($record)
                     'value' => $extra,
                     'short' => false
                 );
+
+                // Add all context items as individual fields
+                foreach ($record['context'] as $var => $val) {
+                    $attachment['fields'][] = array(
+                        'title' => $var,
+                        'value' => $val,
+                        'short' => true
+                    );
+                }
```

Comment:
I am ok with this but I think it should be in line with the Extra stuff, and both should actually follow the useShortAttachment flag. If it's on both context and extra should be like lines 161-171 above, and if it's false both should be like you have it for the context. And if extra|context are empty no attachment should be created IMO, even in useShortAttachment mode.